src/ConfigurationFactory.php

```diff
@@ -171,7 +171,7 @@ protected function getDefaultConfig($configKey)
                 'default_lifetime' => 3600,
                 'default_lock_lifetime' => 60,
                 'file_lock_region_directory' => '',
-                'regtions' => [],
+                'regions' => [],
```

Comment:
is this a BC break @DASPRiD ?

Reply:
I'd say no, this as a bug which is fine to be fixed.